### PR TITLE
Fix set point and also change IDs to String

### PR DESCRIPTION
**Pull request by foxy82**
Hey,

So in this commit I've changed all IDs to be Strings - we receive it as a String never have to do any maths on them and with the weird formatting that is happening I think this is safest. 

I've also fixed the set point and duplicated the channel - one is read only so nice and easy to see in the PaperUI the other is read/write but not implemented yet.

### CODE REVIEW

**Comment by Nebula83** [comment]
Cool beans! Btw, since setting mode and reading temps are working now, could we initiate a PR on the main repo? I think it is V1 worthy?

**Comment by foxy82** [comment]
Yeah I think so - although we probably want to hide the setting temp channel (set point) whilst we figure out how best to implement it.

This binding now does all the original did except weather which isn't in the new V2 api.

**Comment by foxy82** [comment]
I've commented out the set point channel - not tested it but it should work.

**Comment by Nebula83** [comment]
Yeah that would be best I think. About the weather; I can seem to locate it in the V2 API as well. Is that important to you? I am using the weather (wunderground) binding and I'm pretty happy with it

**Comment by Nebula83** [comment]
Hi @foxy82, I haven't received a reply yet on my questions. Maybe your away for a bit, no problem. I check on the issues I found myself and test the available channel so that I can raise a PR on the main repo.

File: addons/binding/org.openhab.binding.evohome/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -30,27 +30,34 @@
...
         <item-type>String</item-type>
-        <label>Setpoint Status</label>
-        <description>Thermostat setpoint status</description>
+        <label>Set point status</label>
+        <description>Thermostat set point status</description>
         <category>heating</category>
-        <state readOnly="true"/>
+        <state pattern="%s" readOnly="true">
+            <options>
+                <option value="PermanentOverride">Permanent Override</option>
```
Comment (Nebula83) [inline addons/binding/org.openhab.binding.evohome/ESH-INF/thing/channels.xml]: Spaces needed to be removed. I fixed it in my last commit (pushed)

**Comment by foxy82** [comment]
Sorry I missed this. Thanks. Although I wonder if I should leave as text as
it is read only...

**Comment by Nebula83** [comment]
No problem, don't worry. 
I'd leave it as `option`. Paper UI now shows `Permanent Override` instead of `PermanentOverride`. It separates presentation from business view, which I like.

**Pull request merged**